packages/connector-twilio-sms/README.md

```diff
@@ -1,2 +1,73 @@
-### Twilio SMS README
-placeholder
+# Twilio short message service connector
+
+Twilio provides programmable communication tools for making and receiving phone calls, sending and receiving text messages, and performing other communication functions using its web service APIs. We can take advantage of its sending text message function to send a "verification code" (or in other words, passcode).
+
+## Register and sign in Twilio account
+
+Go to https://www.twilio.com to register a new account and sign in to Twilio afterward. You may skip the registration step if you've already got an account.
+
+## Set up sender's phone numbers
+
+Go to the [Twilio console page](https://console.twilio.com/) and sign in with your Twilio account.
+
+Switch to _Develop_ tab on the top of the sidebar, and then unfold _Phone Numbers_ and _Messaging_.
+
+Click _Buy a number_ under _Phone Numbers_ -> _Manage_, and you can buy phone number(s) as your sender's phone number.
+
+Go to _Messaging_ -> _Services_ and click _Create Messaging Service_ button on the top-right corner of the panel.
+    1. Name your messaging service and type in the name in _Messaging Service friendly name_ field. Choose the proper purpose, in our case, is `Notify my users`.
+    2. Go to _Add Senders_, choose `Phone Number` as _Sender Type_, and continue. Select the sender you want to link to the messaging service that is currently being created and move on by clicking _Add Phone Numbers_. Need to notice that a phone number (or sender) can only be bound to one messaging service.
```

Comment:
```suggestion
    2. Following the next step, choose `Phone Number` as _Sender Type_, and link the phone number we just claimed to this service. (Note: Each phone number can only be linked with one messaging service)
```

Reply:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/36393111/176342583-7b36b588-f427-4949-b16a-1dce7a7e3ee2.png">
 
 skipped?

Reply:
Also need the summary sentence, sth like, `goes to the next step once we have your SMS service set up`